src/Tools/dotnet-user-jwts/src/Resources.resx

```diff
@@ -0,0 +1,252 @@
+    <value>Claims to add to the JWT. Specify once for each claim in the format "name=value".</value>
+  </data>
+  <data name="CreateCommand_Confirmed" xml:space="preserve">
+    <value>New JWT saved with ID '{id}'.</value>
+  </data>
+  <data name="CreateCommand_Description" xml:space="preserve">
+    <value>Issue a new JSON Web Token</value>
+  </data>
+  <data name="CreateCommand_ExpiresOnOption_Description" xml:space="preserve">
+    <value>The UTC date &amp; time the JWT should expire in the format 'yyyy-MM-dd [[[[HH:mm]]:ss]]'. Defaults to 6 months after the --not-before date. Do not use this option in conjunction with the --valid-for option.</value>
+  </data>
+  <data name="CreateCommand_InvalidClaims_Error" xml:space="preserve">
+    <value>Malformed claims supplied. Ensure each claim is in the format "name=value".</value>
+  </data>
+  <data name="CreateCommand_InvalidDate_Error" xml:space="preserve">
+    <value>The date provided for '{type}' could not be parsed. Dates must consist of a date and can include an optional timestamp.</value>
+  </data>
+  <data name="CreateCommand_InvalidPeriod_Error" xml:space="preserve">
+    <value>The period provided for '{0}' could not be parsed. Ensure you use a format like '10d', '22h', '45s' etc.</value>
+  </data>
+  <data name="CreateCommand_IssuerOption_Description" xml:space="preserve">
+    <value>The issuer of the JWT. Defaults to 'dotnet-user-jwts'.</value>
+  </data>
+  <data name="CreateCommand_NameOption_Description" xml:space="preserve">
+    <value>The name of the user to create the JWT for. Defaults to the current environment user.</value>
+  </data>
+  <data name="CreateCommand_NoAudience_Error" xml:space="preserve">
+    <value>Could not determine the project's HTTPS URL. Please specify an audience for the JWT using the --audience option.</value>
+  </data>
+  <data name="CreateCommand_NotBeforeOption_Description" xml:space="preserve">
+    <value>The UTC date &amp; time the JWT should not be valid before in the format 'yyyy-MM-dd [[HH:mm[[:ss]]]]'. Defaults to the date &amp; time the JWT is created.</value>
+  </data>
+  <data name="CreateCommand_RoleOption_Description" xml:space="preserve">
+    <value>A role claim to add to the JWT. Specify once for each role.</value>
+  </data>
+  <data name="CreateCommand_SchemeOption_Description" xml:space="preserve">
+    <value>The scheme name to use for the generated token. Defaults to 'Bearer'.</value>
+  </data>
+  <data name="CreateCommand_ScopeOption_Description" xml:space="preserve">
+    <value>A scope claim to add to the JWT. Specify once for each scope.</value>
+  </data>
+  <data name="CreateCommand_ValidForOption_Description" xml:space="preserve">
+    <value>The period the JWT should expire after. Specify using a number followed by a period type like 'd' for days, 'h' for hours, 'm' for minutes, and 's' for seconds, e.g. '365d'. Do not use this option in conjunction with the --expires-on option.</value>
```

Comment:
I found this sentence a bit confusing, as at first I thought period meant `.`. Maybe say "duration" instead?

Reply:
Fixed.